docs/reference/ccr/getting-started.asciidoc

```diff
@@ -154,16 +165,23 @@ remote cluster.
 ====
 
 [[ccr-getting-started-leader-index]]
-==== Create a leader index
-To create a leader index, access {kib} on your _remote_ cluster and go to
-*Management > Dev Tools*.
+==== Choose a leader index
```

Comment:
"Choose" seems like a strange choice. :-) In practice, you know what you want to replicate. I think I'd be inclined to drop this section and treat having at least one index on the remote cluster as a prereq.